packages/vuetify/src/components/VMenu/VMenu.ts

```diff
@@ -249,6 +263,15 @@ export default baseMixins.extend({
         this.closeOnClick &&
         !this.$refs.content.contains(target)
     },
+    genActivatorListeners () {
+      const listeners = Menuable.options.methods.genActivatorListeners.call(this)
+
+      if (!this.disableKeys) {
+        listeners.keydown = this.onKeyDown
+      }
+
+      return listeners
+    },
```

Comment:
```
genActivatorListeners () {
  return {
	...Menuable.options.methods.genActivatorListeners.call(this),
    ...(this.disableKeys ? {} : {
      keydown: this.onKeyDown
	})
  }
},
```

Reply:
Imho it's not more performant or readable than the original version